File: CS/Form1.cs
using DevExpress.DashboardCommon;
using DevExpress.DashboardWin;
using DevExpress.XtraCharts;
using DevExpress.XtraReports.UI;
using PieTotalExtension;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace DesignerSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            PieTotalModule extension = new PieTotalModule();
            extension.Attach(dashboardDesigner1);
            dashboardDesigner1.LoadDashboard(@"..\..\Data\dashboard.xml");
            dashboardDesigner1.AsyncMode = true;

        }
    }
}

File: VB/DesignerSample/Form1.vb
Imports DevExpress.DashboardCommon
Imports DevExpress.DashboardWin
Imports DevExpress.XtraCharts
Imports DevExpress.XtraReports.UI
Imports PieTotalExtension
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms

Namespace DesignerSample
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			Dim extension As New PieTotalModule()
			extension.Attach(dashboardDesigner1)
			dashboardDesigner1.LoadDashboard("..\..\Data\dashboard.xml")

		End Sub
	End Class
End Namespace
